home/src/main/res/navigation/home_navigation.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/home_navigation"
+    app:startDestination="@id/homeFragment"
+    tools:ignore="UnusedNavigation">
+
```

Comment:
not a huge deal but it's nice to have a consistency to formatting as you go. lots of extra line breaks here and there that can be cleaned up either manually or with a tool like ktlint